estate/models/estate_property.py

```diff
@@ -0,0 +1,22 @@
+from odoo import fields, models
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Estate property"
+
+    name = fields.Char("Name", required=True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date("Available From", copy=False, default=fields.Date.today())
+    expected_price = fields.Float("Expected Price", required=True)
+    selling_price = fields.Float("Selling Price", readonly=True, copy=False)
+    bedrooms = fields.Integer("Bedrooms", default=2)
+    living_area = fields.Integer("Living Area (sqm)")
+    facades = fields.Integer("Facades")
+    garage = fields.Boolean("Garage")
+    garden = fields.Boolean("Garden")
+    garden_area = fields.Integer("Garden Area (sqm)")
+    garden_orientation = fields.Selection(
+        string='Type',
+        selection=[('north', 'North') ,('south', 'South'), ('east', 'East'), ('west', 'West')],
```

Comment:
In python, we try to follow the following convention for quoting strings:
- single quote for technical terms
- double quotes for displayed texts

=> `('north', "North")`

Let's try to apply it during this exercise, even though the existing codebase contains many counter examples.